### PR TITLE
SR86x lockin: get values of many parameters at once

### PR DESCRIPTION
**Pull request by astafan8**
Adds the possibility to return multiple measured values at the same time, `get_values` by implementing corresponding visa commands. Also, add data channels, `data_channels`, which also allow to return 4 values of parameters assigned to them at once.

ToDo:
- [x] final testing on device
- [x] update example notebook

### CODE REVIEW

**Comment by astafan8** [comment]
@QCoDeS/core can someone have a quick look? :)

File: qcodes/instrument_drivers/stanford_research/SR86x.py [hunk 1]
```diff
@@ -918,3 +1017,95 @@ def _set_sensitivity(self, s):
...
+                requested; valid names can be found in `PARAMETER_NAMES`
+                attribute of the driver class
+
+        Returns:
+            a tuple of 2 or 3 floating point values
+
+        """
+        if not 2 <= len(parameter_names) <= 3:
+            raise KeyError(
+                'It is possible to request values of only 2 or 3 parameters '
```
[the remark this answers is not in the document]
Comment (astafan8) [reply qcodes/instrument_drivers/stanford_research/SR86x.py]: true

File: qcodes/instrument_drivers/stanford_research/SR86x.py [hunk 2]
```diff
@@ -918,3 +1017,95 @@ def _set_sensitivity(self, s):
...
+                'at a time.')
+
+        for name in parameter_names:
+            if name not in self.PARAMETER_NAMES:
+                raise KeyError(f'{name} is not a valid parameter name. Refer '
+                               f'to `_PARAMETER_NAMES` for a list of valid '
```
[the remark this answers is not in the document]
Comment (astafan8) [reply qcodes/instrument_drivers/stanford_research/SR86x.py]: oops. you're right.

**Comment by astafan8** [comment]
@jenshnielsen Yes, i did test (modulo some corner cases that i tested before final refactoring, my bad...). that is how the example notebook was updated. once CI passes, i will merge.
